### PR TITLE
feat: cards component

### LINKED ISSUE #7: Card component
### Cards
- [x] Vertical card
- [x] Horizontal card
- [x] Cards with badges
- [x] Cards with text overlay
- [x] Cards with shadow
- [x] Text only cards
- [x] Cards with dismiss

### PR DESCRIPTION
**Pull request by siddhigate**
Cards:
- Vertical card
- Card with shadow
- Card with text overlay
- Card with badge
- Horizontal card
- Text only card
- Card with dismiss

**Preview:** https://upsleek-css-component-library-git-cards-siddhigate.vercel.app/

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -71,3 +71,14 @@
   height: 1.5rem;
   min-width: 1.5rem;
 }
+
+.badge-pill {
+  background-color: var(--clr-primary);
+  color: white;
```
Comment (ZuberDunge) [inline css/components/badge.css]: suggestion : you can use variables for color

**Review by ZuberDunge — APPROVED** [review]
Code looks fine. good to go.

File: css/components/badge.css (hunk 1 above)
Comment (siddhigate) [reply css/components/badge.css]: Done. Thanks for your suggestion!

**Review by Manju-sree — APPROVED** [review]
very well structured code.. good to merge

**Pull request merged**